Source/DataModels/Cubemap/Cubemap.cpp

```diff
@@ -45,6 +46,7 @@ Cubemap::~Cubemap()
 Cubemap::Cubemap(std::shared_ptr<ResourceCubemap> cubemapRes) : cubemapRes(cubemapRes)
```

Comment:
```suggestion
Cubemap::Cubemap(std::shared_ptr<ResourceCubemap> cubemapRes) : cubemapRes(cubemapRes), intensity(1.f)
```
its more clean like this, but its the same so np